societes/france.json

```diff
@@ -6243,6 +6289,54 @@
                 }
             ]
         },
+        {
+            "name": "Ironova",
+            "url": "https://www.ironova.com",
+            "description": "La capture émotionnelle arrive sous la forme d’un bracelet",
+            "locations": [
+                {
+                    "name": "",
+                    "postal_address": "45 Rue Frédéric Joliot Curie, Marseille, France",
+                    "gps_coordinates": "43.3439196/5.4329399"
+                }
+            ]
+        },
+        {
+            "name": "Ironova",
+            "url": "https://www.ironova.com",
+            "description": "La capture émotionnelle arrive sous la forme d’un bracelet",
+            "locations": [
+                {
+                    "name": "",
+                    "postal_address": "45 Rue Frédéric Joliot Curie, Marseille, France",
+                    "gps_coordinates": "43.3439196/5.4329399"
+                }
+            ]
+        },
+        {
+            "name": "Iskn",
+            "url": "http://www.isketchnote.com",
+            "description": "Writing devices",
+            "locations": [
+                {
+                    "name": "",
+                    "postal_address": "52 Cours Jean Jaurès, 38000 Grenoble",
+                    "gps_coordinates": "45.18760/5.71854"
+                }
+            ]
+        },
+        {
+            "name": "MedTech spécialisée dans l'autotransfusion sanguine",
```

Comment:
Small mistake, description field in the name.

Reply:
fixed